### PR TITLE
add simpleyaml and simplexml driver support

### PR DESCRIPTION
**Pull request by dominikzogg**

### CODE REVIEW

**Comment by dominikzogg** [comment]
@jmikola? do i need to change anything?

**Comment by dominikzogg** [comment]
@jmikola ping

File: docs/en/reference/xml-mapping.rst [hunk 1]
```diff
@@ -63,6 +63,28 @@ of the constructor, like this:
     $driver = new XmlDriver(array('/path/to/files'));
     $config->setMetadataDriverImpl($driver);
 
+Simplified XML Driver
+~~~~~~~~~~~~~~~~~~~~~
+
+The Symfony project sponsored a driver that simplifies usage of the XML Driver.
+The changes between the original driver are:
+
+1. File Extension is .orm.xml
```
Comment (malarzm) [inline docs/en/reference/xml-mapping.rst]: File extension is mongodb-odm.xml according to next line

File: docs/en/reference/yml-mapping.rst [hunk 2]
```diff
@@ -43,6 +43,28 @@ of the constructor, like this:
     $driver = new YamlDriver(array('/path/to/files'));
     $config->setMetadataDriverImpl($driver);
 
+Simplified YAML Driver
+~~~~~~~~~~~~~~~~~~~~~~
+
+The Symfony project sponsored a driver that simplifies usage of the YAML Driver.
+The changes between the original driver are:
+
+- File Extension is .orm.yml
```
Comment (malarzm) [inline docs/en/reference/yml-mapping.rst]: Same here

File: tests/Doctrine/ODM/MongoDB/Tests/Mapping/Symfony/XmlDriverTest.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+use Doctrine\ODM\MongoDB\Mapping\Driver\SimplifiedXmlDriver;
+
+/**
+ * @group DDC-1418
+ */
+class XmlDriverTest extends AbstractDriverTest
+{
+    protected function getFileExtension()
+    {
+        return '.mongodb-odm.xml';
```
Comment (malarzm) [inline tests/Doctrine/ODM/MongoDB/Tests/Mapping/Symfony/XmlDriverTest.php]: Isn't it better to reuse const from driver?

File: tests/Doctrine/ODM/MongoDB/Tests/Mapping/Symfony/YamlDriverTest.php [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+use Doctrine\ODM\MongoDB\Mapping\Driver\SimplifiedYamlDriver;
+
+/**
+ * @group DDC-1418
+ */
+class YamlDriverTest extends AbstractDriverTest
+{
+    protected function getFileExtension()
+    {
+        return '.mongodb-odm.yml';
```
Comment (malarzm) [inline tests/Doctrine/ODM/MongoDB/Tests/Mapping/Symfony/YamlDriverTest.php]: Same here

File: tests/Doctrine/ODM/MongoDB/Tests/Mapping/Symfony/XmlDriverTest.php (hunk 3 above)
Comment (dominikzogg) [inline tests/Doctrine/ODM/MongoDB/Tests/Mapping/Symfony/XmlDriverTest.php]: this a port of the orm code, how would your suggestion looks like?

Comment (malarzm) [inline tests/Doctrine/ODM/MongoDB/Tests/Mapping/Symfony/XmlDriverTest.php]: Ok, I won't argue with that then :)

**Comment by malarzm** [comment]
@dominikzogg could you squash commits?

**Comment by dominikzogg** [comment]
@malarzm done

**Comment by malarzm** [comment]
Merged manually in https://github.com/doctrine/mongodb-odm/commit/d3e9b50886d330dbd8cf8ad51dd5cea0346b3f80 - thanks @dominikzogg!

**Comment by dominikzogg** [comment]
@malarzm thx for merging

**Pull request closed without merging**